.github/workflows/vm-freebsd.yml

```diff
@@ -10,13 +10,14 @@ on:
 
 jobs:
   build-freebsd:
-    if: false && (github.event_name == 'push' || github.event.pull_request.merged == true)
-    name: FreeBSD ${{ matrix.osversion }} ${{ matrix.abi == '64' && ' ' || matrix.abi }} ${{ matrix.nroff == 'true' && 'no-roff' || ' ' }} ${{ matrix.cflags == '' && ' ' || 'max-Werror' }}
+    if: true && (github.event_name == 'push' || github.event.pull_request.merged == true)
```

Comment:
If I understand, this "false" -> "true" is there to quickly toggle the FreeBSD build on/off?
Having `true && expression` is a bit surprising, but this is also infrastructure configuration meant for maintainers, so good to have a quick toggle

Reply:
Yes, exactly, this was to to speed up the GitHub Actions development cycle a bit. Not sure it's worth keeping around. If there were a more declarative way to temporarily skip some...